src/parsec/util.cpp

```diff
@@ -183,6 +184,8 @@ namespace cbdc::parsec {
                 cfg.m_runner_type = runner_type::evm;
             } else if(val == "lua") {
                 cfg.m_runner_type = runner_type::lua;
+            } else if(val == "py") {
```

Comment:
While I love this, I don't believe this should be in here - part of the other PR you are on AFAIK

Reply:
Good catch, thanks!